### ISSUE TITLE
Install DLC with GUI on Ubuntu 20.04

### ISSUE THREAD

**Issue by kaymvoit** [DeepLabCut/DeepLabCut#1392]
Lots of people seem to report problems with installing DLC with GUI on Ubuntu 20.04. I finally managed, but none of the descriptions really resolved my issues by themselves, I had to resort to multiple resources including some not specific to DLC.

To start with, here is how I got actually being able to start the GUI:
```
conda create -n DLC python=3.7
conda activate DLC
pip install tensorflow==1.15
pip install setuptools==43.0.0
pip install -U -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-16.04 wxPython==4.0.7
pip install 'deeplabcut[gui]'
python -m deeplabcut
```
I ran into two issues on the way. 

1. Getting wxPython to run. This is kind of linked in the documentation, but in addion, I had to downgrade setuptools. After that wxPython installed, but only from the ubuntu-16.04 repo. Without the -f switch, it fails, and the ubuntu-20.04 repo does not have versions <4.1.0
2. Getting tensorflow to work. This si also why I write this as a bug report. The install instructions explicitly say to make a python 3.8 environment, but as far as I found out, tensorflow 1.15 isn't available for python 3.8, while I had no success with tensorflow>=2. Maybe I am missing something here, but I feel that the install instructions should be either fixed, or extended.

**Comment by MMathisLab** [comment]
@kaymvoit did you see the how to here, maybe helpful: https://deeplabcut.github.io/DeepLabCut/docs/recipes/installTips.html#installation-on-ubuntu-20-04-lts

and I fully urge you to try DEEPLABCUT.yaml; I can 100% confirm this works for me on ubuntu 20.04 LTS. http://www.mackenziemathislab.org/s/DEEPLABCUT-znfp.yaml

**Comment by kaymvoit** [comment]
I did see that, but since there was also [this](https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/installation.md#creating-your-own-customized-conda-env-recommended-route-for-linux-ubuntu-centos-mint-etc) and I didn't obviously see how adding a container would solve the issues, I didn't go for that. Mind, I did get it running, I am only trying to help improve the documentation for the next person now.

Except for the version pinning on deeplabcut[gui], the YAML seems to be the same as the one that is shipped with the repository.  Both fail for me due to the wxPython issue, and since it also calls for python 3.8, I don't understand how that would work with tensorflow 1.15. On my system, pip in a python 3.8 environment only offers tensorflow packages starting from 2.0.

```
~ $ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.2 LTS"
```
